lib/image-stream/parse-partitions.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const Bluebird = require('bluebird');
+const MBR = require('mbr');
+const GPT = require('gpt');
+
+const MAX_BYTES = 65536;
+const BLOCK_SIZE = 512;
+const ZERO = 0;
```

Comment:
lol. The intention of disallowing magic numbers is to be able to pick a constant name that better explains what the value means in a particular context. In this case, the number 0 means two different things.

- Its used on `getEfiPart` to get the first value of the filtered array. You can use `_.first` to better showcase the meaning, or you can replace `_.filter` + `_.first` with `_.find`, which is the best way to go IMO

- Its used on the exported function as the initial value of a variable that is increased through the function. In this case, a good variable name would be `LENGTH_INITIAL_VALUE` or something like that